source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/summary/PropertyManagementDetailView.tsx

```diff
@@ -48,7 +48,7 @@ export const PropertyManagementDetailView: React.FC<IPropertyManagementDetailVie
           }
         />
       </SectionField>
-      <SectionField label="Lease/Licensed">
+      <SectionField label="Lease/Licenced">
```

Comment:
sorry to be picky here - **licensed** (as a verb) is the correct work in British/Canadian English. 
Please change to **Licence** (noun) or **Licensed** (verb)

https://ontariotraining.net/word-choice-license-versus-licence/#:~:text=Examples%20(British%20and%20Canadian%20English)&text=(Licence%20is%20a%20noun.),adjectives%2C%20licenced%20may%20be%20used.

Reply:
Updated